.dvc/config

```diff
@@ -1,3 +1,5 @@
+[studio]
+    token = dsat_2i8Sh6ta0qkeGcMxvym5atQycL3drhDYSA99JfvkjKDkkSGV
```

Comment:
What is this token and should it be publicly accessible?

Reply:
yeah it should be there, but not useful yet